doc/tracking_output.rst

```diff
@@ -0,0 +1,11 @@
+Tracking output
+-------------------------
+
+Tracking outputs a `pandas` dataframe with additional variables in addition to the variables output by Feature Detection (see :doc:`feature_detection_output`). The additional variables added by tracking, with column names listed in the `Variable Name` column, are described below with units. 
```

Comment:
Is the output in the same dataframe as the feature detection output, or a different dataframe? Can you make that more clear?

Also, probably do not need the word "additional" since you have "in addition to" a few words later.